packages/hardhat-astar-plugin/test/project.test.ts

```diff
@@ -0,0 +1,46 @@
+// tslint:disable-next-line no-implicit-dependencies
+import { assert } from "chai";
+import path from "path";
+
+import { ExampleHardhatRuntimeEnvironmentField } from "../src/ExampleHardhatRuntimeEnvironmentField";
+
+import { useEnvironment } from "./helpers";
+
+describe("Integration tests examples", function () {
+  describe("Hardhat Runtime Environment extension", function () {
+    useEnvironment("hardhat-project");
+
+    it("Should add the example field", function () {
+      assert.instanceOf(
+        this.hre.example,
+        ExampleHardhatRuntimeEnvironmentField
+      );
+    });
+
+    it("The example filed should say hello", function () {
+      assert.equal(this.hre.example.sayHello(), "hello");
+    });
+  });
+
+  describe("HardhatConfig extension", function () {
+    useEnvironment("hardhat-project");
+
+    it("Should add the newPath to the config", function () {
+      assert.equal(
+        this.hre.config.paths.newPath,
+        path.join(process.cwd(), "asd")
+      );
+    });
+  });
+});
+
+describe("Unit tests examples", function () {
+  describe("ExampleHardhatRuntimeEnvironmentField", function () {
+    describe("sayHello", function () {
```

Comment:
These should be changed to use `sdk-core` API and test the evm conversion tools.